src/main/java/org/jitsi/jibri/service/JibriService.kt

```diff
@@ -0,0 +1,27 @@
+package org.jitsi.jibri.service
+
+import org.jitsi.jibri.util.StatusPublisher
+
+enum class JibriServiceStatus {
+    FINISHED,
+    ERROR
+}
+
+typealias JibriServiceStatusHandler = (JibriServiceStatus) -> Unit
+
+/**
+ * Interface implemented by all implemented [JibriService]s.  A [JibriService]
+ * is responsible for an entire feature of Jibri, such as recording a call
+ * to a file or streaming a call to a url.
+ */
+abstract class JibriService : StatusPublisher<JibriServiceStatus>() {
```

Comment:
interface*

Reply:
fixed

Reply:
actually nm, this can't be an interface because it inherits from `StatusPublisher`